tests/test_bevel.py

```diff
@@ -1,85 +1,51 @@
-#!/usr/bin/env python3
 import hashlib
 from pathlib import Path
+from tests.shared import hex_code_check
 
 from cycax.cycad import SheetMetal
+from itertools import combinations
 
 
-def fancy_edge(edge_loc):
-    sheet = SheetMetal(x_size=100, y_size=200, z_size=50, part_no="sheet_bevel")
 
-    sheet.beveled_edge(edge_type="round", side1="FRONT", side2="RIGHT", size=3)
-    sheet.beveled_edge(edge_type="round", side1="FRONT", side2="LEFT", size=3)
-    sheet.beveled_edge(edge_type="round", side1="BACK", side2="LEFT", size=3)
-    sheet.beveled_edge(edge_type="round", side1="BACK", side2="RIGHT", size=3)
+def fancy_edge(tmp_path: Path, edge_type:str, part_no:str):
+    sheet = SheetMetal(x_size=100, y_size=200, z_size=50, part_no=part_no)
 
-    sheet.beveled_edge(edge_type="round", side1="TOP", side2="FRONT", size=3)
-    sheet.beveled_edge(edge_type="round", side1="BOTTOM", side2="FRONT", size=3)
-    sheet.beveled_edge(edge_type="round", side1="BACK", side2="TOP", size=3)
-    sheet.beveled_edge(edge_type="round", side1="BACK", side2="BOTTOM", size=3)
+    edges = combinations(("LEFT", "RIGHT", "FRONT", "BACK", "TOP", "BOTTOM"), 2)
+    for edge in edges:
```

Comment:
This is fine for now. Please make a Jira to catch the exception.

So ... your testing all the use cases except the problematic ones. Your the one steeling all the cashews out of the mix nuts packet. 
Use `with pytest.raises(Exception)` and catch the exception, 
I like the use of elif and tuples, thats very good this could be further improved with
`if edge in ( ("LEFT", "RIGHT"), ("TOP", "BOTTOM"), .......` but first catch the errors from beveld_edge and then see if you need to improve.

It might be good to raise custom exceptions when cycax is tasked with an impossible task.

Reply:
https://tsolo.atlassian.net/browse/SS-178